### PR TITLE
Rewrite setup to match conda-forge version

### PR DESCRIPTION
**Pull request by willirath**
This is relative to `v1.0` for now.

### CODE REVIEW

**Comment by willirath** [comment]
@erikvansebille @delandmeterp @Simnator101 Tests are successful with this version.  So from by POV, this is ready to go.

Can somebody check if this also works on OSX and Windows?

**Comment by willirath** [comment]
Note that in the conda-forge version, we cannot use `setuptools_scm` (which derives the version number for the installation from the Git ref).  Hence, this version differs from the one used on conda-forge here: <https://github.com/OceanParcels/parcels/pull/282/files#diff-2eeaed663bd0d25b7e608891384b7298R13>

**Comment by willirath** [comment]
<https://github.com/OceanParcels/parcels/pull/282/files#diff-caf2a6b8f4947d018f68893c695b5202> is my stab at a (quite radical) rewrite of the whole installation section.

The instructions now cover three classes of users:

1. People who just want to use the stable and realeased Parcels.  They
get a fully working Python setup with Parcels from conda forge.

2. People who want to get a fully working Setup but who aim for an
unreleased version.  (To use a bug-fixed version, for example.)

3. Devs who want to maintain in full control of their environment and
who want to be able to edit the installed package without re-installing.

(Class 2. contains all those that benefit from new features that they might have contributed to (by filing an issue or adding a feature request) and who probably are eager to have a quick way of acutally using the fixed code.)

**Comment by willirath** [comment]
@erikvansebille I'm not really fluent in RST. So my changes are a bit lazy in references and links.

File: docs/index.rst [hunk 1]
```diff
@@ -80,65 +80,72 @@ The best way to get started with Parcels is to have a look at the Jupyter notebo
...
+============================================
 
-An alternative, more advanced option is to use an existing netCDF installation and
-C compiler, ``git clone`` the `master branch of Parcels
-<https://github.com/OceanParcels/parcels>`_
-and ``pip install`` the dependencies in the `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment.yml>`_ file.
+There might be cases where want to install a version of Parcels that has not been released yet.  (Perhaps, if you want to use a bleeding-edge feature which already is included on Github, but not in the conda-forge package.)
 
-.. _installing-windows:
+Then, just after Step 2. of :ref:`installing-parcels` above, remove the conda-forge package again, and use Pip to install Parcels from Github::
```
Comment (erikvansebille) [inline docs/index.rst]: Do we need to say how to remove the parcels package from conda?

File: docs/index.rst [hunk 2]
```diff
@@ -80,65 +80,72 @@ The best way to get started with Parcels is to have a look at the Jupyter notebo
...
-        conda env create -f environment_win.yml -n py2_parcels
-        activate py2_parcels
-        pip install git+https://github.com/OceanParcels/parcels.git@master
+Parcels depends on a working Python installation, a netCDF installation, a C compiler, and various Python packages.  If you prefer to maintain an own Python installation providing all this, ``git clone`` the `master branch of Parcels <https://github.com/OceanParcels/parcels>`_ and ``pip install`` the dependencies in the `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment.yml>`_ file.
 
-5. The next time you start the ``Anaconda Prompt`` and want to work with Parcels, activate the environment with::
+Then, install Parcels in an `editable way <https://pip.pypa.io/en/stable/reference/pip_install/?highlight=editable#cmdoption-e>` by running::
 
-        activate py2_parcels
+    pip install -e .
```
Comment (erikvansebille) [inline docs/index.rst]: Do we need to say anything about $PYTHONPATH here? That was what @AlexCK has real problems with on Slack

File: docs/index.rst [hunk 3]
```diff
@@ -80,65 +80,72 @@ The best way to get started with Parcels is to have a look at the Jupyter notebo
...
-        conda env create -f environment_win.yml -n py2_parcels
-        activate py2_parcels
-        pip install git+https://github.com/OceanParcels/parcels.git@master
+Parcels depends on a working Python installation, a netCDF installation, a C compiler, and various Python packages.  If you prefer to maintain your own Python installation providing all this, ``git clone`` the `master branch of Parcels <https://github.com/OceanParcels/parcels>`_ and ``pip install`` the dependencies in the `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment.yml>`_ file.
 
-5. The next time you start the ``Anaconda Prompt`` and want to work with Parcels, activate the environment with::
+Then, install Parcels in an `editable way <https://pip.pypa.io/en/stable/reference/pip_install/?highlight=editable#cmdoption-e>`_ by running::
 
-        activate py2_parcels
+    pip install -e .
```
Comment (erikvansebille) [inline docs/index.rst]: Do we need to say anything about $PYTHONPATH here? That was what @alexck has real problems with on Slack

File: docs/index.rst [hunk 4]
```diff
@@ -80,65 +80,72 @@ The best way to get started with Parcels is to have a look at the Jupyter notebo
...
+============================================
 
-An alternative, more advanced option is to use an existing netCDF installation and
-C compiler, ``git clone`` the `master branch of Parcels
-<https://github.com/OceanParcels/parcels>`_
-and ``pip install`` the dependencies in the `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment.yml>`_ file.
+There might be cases where you want to install a version of Parcels that has not been released yet.  (Perhaps, if you want to use a bleeding-edge feature which already is included on Github, but not in the conda-forge package.)
 
-.. _installing-windows:
+Then, just after step 2 of :ref:`installing-parcels` above, remove the conda-forge package again, and use Pip to install Parcels from Github::
```
Comment (erikvansebille) [inline docs/index.rst]: Do we need to say how to remove the parcels package from conda?

Comment (willirath) [reply docs/index.rst]: Oh. Yes, I forgot to add a `conda remove parcels`.

Note that unlike package managers like apt or brew, conda does not remember where all the dependencies came from and really just removes the parcels package here.

File: docs/index.rst (hunk 3 above)
Comment (willirath) [reply docs/index.rst]: No. Pip installs parcels to some standard path for the env that was activated while running the command.

Comment (willirath) [reply docs/index.rst]: I'm not sure how these virtualenvs work, but I'd be surprised if after a pip install there's any manual adaptation of the path necessary.

**Pull request merged**